lib/pos_fee_estimator.go

```diff
@@ -675,8 +677,13 @@ func (posFeeEstimator *PoSFeeEstimator) estimateFeeRateNanosPerKBGivenTransactio
 		}
 	}
 	globalMinFeeRate, _ := txnRegister.minimumNetworkFeeNanosPerKB.Uint64()
-	if len(txns) == 0 {
-		// If there are no txns in the transaction register, we simply return the minimum network fee.
+	// Compute the congestion threshold. If our congestion factor is 100% (or 10,000 bps),
+	// then congestion threshold is simply max block size * numPastBlocks
+	congestionThreshold := (congestionFactorBasisPoints * maxSizeOfNumBlocks) / MaxBasisPoints
```

Comment:
Do we need to check overflow? I think it's fine because they'll never break the uint64 right?

Reply:
it'll never overflow - it's safe.